messages/delete/edge_application/messages.go

```diff
@@ -11,4 +11,5 @@ var (
 	CascadeSuccess   = "Remote resources deleted successfully"
 	FlagId           = "Unique identifier of the Edge Application"
 	AskInput         = "Enter the ID of the Edge Application you wish to delete:"
+	CONFDIRFLAG      = "Relative path to where your custom azion.json and args.json files are stored"
```

Comment:
non-blocking: Porque a variável está em CAPS?

Reply:
I ended-up copying from another command, and eventually we want all the message variables to be constants